test/service/AggregateTransactionService.spec.ts

```diff
@@ -188,12 +188,9 @@ describe('AggregateTransactionService', () => {
         );
 
         const signedTransaction = aggregateTransaction.signTransactionWithCosignatories(account1, [account2], generationHash);
-        aggregateTransactionService
-            .isComplete(signedTransaction)
-            .toPromise()
-            .then((isComplete) => {
-                expect(isComplete).to.be.true;
-            });
+        return firstValueFrom(aggregateTransactionService.isComplete(signedTransaction)).then((isComplete) => {
```

Comment:
did you intentionally add return?

Reply:
Yes, there are bugs in the tests. If return or await is not added, the async assertion is excluded from the test.

Let me add them when I see them missing

Reply:
so, that means all the tests without returns are broken, doesn't it?

Reply:
Not all but then ones that did not 'await' or 'return'. I have fixed those.